draft-ferrieuxhamchaoui-quic-lossbits.md

```diff
@@ -422,7 +422,9 @@ Value: 0x1057 (if this document is approved)
 
 Parameter Name: loss_bits
 
-Specification: ..............
+Specification: Indicates that the endpoint supports loss bits. An endpoint that
```

Comment:
I don't think we need to recap how the transport parameter is used here -- it is done in the _Transport Parameter_ section.  We do something simple instead:

> Specification: Indicates that the endpoint supports receiving or sending loss bits.

Reply:
I was basing this on the DATAGRAM draft (https://tools.ietf.org/html/draft-pauly-quic-datagram-05#section-7).

> Specification:  Indicates that the connection should enable support
      for unreliable DATAGRAM frames.  An endpoint that advertises this
      transport parameter can receive datagrams frames from the other
      endpoint, **up to and including the length in bytes provided in the
      transport parameter.**

Reply:
I was using the [1wd draft](https://tools.ietf.org/html/draft-huitema-quic-1wd-00#section-6) for reference:

```
   Specification: Indicates that the connection should use TimeStamped
   ACK frames
```

Reply:
@dtikhonov Well, 1wd draft defines a zero-length transport parameter. The DATAGRAM draft defines an integer parameter and asks IANA to mention the meaning of the parameter value in the registry.  We also define an integer parameter...

Reply:
OK, it seems your preference is stronger than mine -- looks good to me.

Reply:
Maybe a little stronger, though this is certainly something that could be trivially changed by the WG with 0 consequences to the design. :)